qgit status

```diff
@@ -0,0 +1,329 @@
+[1mdiff --git i/chain/jsonrpc-primitives/src/types/mod.rs w/chain/jsonrpc-primitives/src/types/mod.rs[m
```

Comment:
What is this file for? Is it supposed to be committed?

Reply:
removed